DiscordBot.js

```diff
@@ -15,45 +15,216 @@ class DiscordBot {
     
         this.client.on('message', msg => {
 
-            let prefix = "§";
-            let channelFilePath = "config/discordchannels.json";
+            let prefix = "$";
+
+            const channelsFilePath  = "config/channels.json";
+            const channelsFile      = JSON.parse(fs.readFileSync(channelsFilePath));
+            const coursesFilePath   = "config/courses.json";
+            const coursesFile       = JSON.parse(fs.readFileSync(coursesFilePath));
+
+            let respondContent = "";
+            
+            const addColor      = "#00ff00";
+            const deleteColor   = "#ff0000";
+            const messageColor  = "#555555";
 
             if (msg.content.startsWith(prefix)) {
+                this.botLog(msg);
+                let args = msg.content.split(" "); //split in arguments
                 switch(true) {
-                    case msg.content.startsWith(prefix + "help"):
-                        msg.reply("List of commands: \n" +
-                            "§ping -> should send back a Pong! \n" +
+//help
+                    case msg.content == prefix + "help" || msg.content == prefix:
+                        this.sendList(msg.channel.id, "List of commands:", ["Command", "Description", "Shortcut"], 
+                            [
+                                [
+                                    "§help", 
+                                    "§ping", 
+                                    "§refresh", 
+                                    "§listChannels", 
+                                    "§addChannel name id/this", 
+                                    "§rmChannel id/this", 
+                                    "§listCourses", 
+                                    "§list name", 
+                                    "§resend filename"
```

Comment:
Dies kann in eine seperate .json ausgelagert werden, da der Inhalt (bzw. Text) komplett statisch ist. Außerdem sollte das präfix dynamisch hinzugefügt werden falls es sich ändert.